pakiti-client

```diff
@@ -369,9 +368,41 @@ sub encrypt_report ($) {
 # send a formatted report
 #
 
+sub https {
+    my ($url, $report) = @_;
+    my($ua, $response);
+    eval { require LWP::UserAgent; LWP::UserAgent->import; };
+    if ($@) {
+        if ($Option{"disable-tls-checks"}) {
+            $response = `curl -sik -o /dev/null -w "%{http_code}" -X POST $url -d "$report"`;
+        } else {
+            $response = `curl -si -o /dev/null -w "%{http_code}" -X POST $url -d "$report"`;
+        }
+        if ($response == "200") {
```

Comment:
in order to maintain the readability, I'd stick with the same pattern as in the other if branch (and the original code), i.e., ($response == "200") or die